WordPress/Classes/Utility/FormattableContent/Actions/FormattableContentAction.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
+import WordPressShared
```

Comment:
Is it necessary to import Foundation?

I don't see any other changes in the diff, so if it worked before, my guess is that it's not necessary at this time, although it might be in the future if the code uses a Foundation type and it's imported via bridging header.

If it's not necesary:

```suggestion
import WordPressShared
```